src/Troubleshooting/index.md

```diff
@@ -95,77 +54,34 @@ There's no known fix (as of October 14th, 2023), but the following workarounds m
-
----
-
-## Studio Issues
-
-Known Roblox Studio issues:
-
-### Stuck on the splash screen
-
-Studio is currently broken with 96 DPI rendering. Set the DPI to 97 (already done by default) by running `vinegar exec winecfg` to open the Wine configuration, and then head to the Graphics tab. There should be a DPI option at the bottom of the page.
-
-Other DPI values work, as long as they're **NOT** 96. Note that values not divisible by 96 will make the interface blurrier, and that the game world view becomes blurrier as DPI increases. This is a limitation with Wine's X11 implementation.
-
-### Unable to log in - WebView2 error
-
-WebView2 is currently not supported, so there's no way to log in directly inside of studio.
+If you get a black login window you can add `webview = ""` under `[studio]` in your Vinegar config or you can just run `pkill -f webview` when the black window appears and you should be prompted to login with your browser.
 
 Follow the on-screen prompt and click "Log in via browser". This will open an authentication website in your browser. Once you're done authenticating, you might be prompted to allow the use of the `roblox-studio-auth` protocol. Make sure to say "yes".
 
+If clicking on the button doesn't do anything you may need to install the `xdg-desktop-portal-gtk` package, the name may differ on some distributions.
+
 If all steps were followed correctly, studio should automatically log into your account.
 
 If you're still unable to log in, try changing your DNS to a viable alternative such as `1.1.1.1` or `8.8.8.8`. Studio authentication is known to be broken with certain DNS providers. 
 
 ### "Your GPU is incompatible" error
 
-This issue is most commonly caused by the Wine build not having the Vulkan ChildWindow patch applied. There are a couple solutions:
-
-- Using the [Vinegar flatpak](https://vinegarhq.org/Installation/guides/flatpak.html), which includes the ChildWindow patch;
+Make sure your drivers are installed correctly, if that doesn't help you can:
 
-- Using a Wine build that has the ChildWindow patch enabled (such as [Wine GE](https://github.com/GloriousEggroll/wine-ge-custom));
+- Use the [Vinegar flatpak](https://vinegarhq.org/Installation/guides/flatpak.html)
 
-- Disabling DXVK in the Vinegar configuration.
+- Disable DXVK in the configuration
 
 ### "Necessary graphics drivers not installed" error
 
 This issue can be fixed by switching renderer to Vulkan, instructions on how to do so are located [here](https://vinegarhq.org/Configuration/index.html).
 
-### Buggy QT fonts
-
-This is a known issue with WineD3D. Enabling DXVK in the configuration file should solve this issue.
-
 ### Flickering widgets and plugin windows
 
-This is a known issue with WineD3D. Enabling DXVK in the configuration file should solve this issue.
+Disabling DXVK in the configuration should solve this issue.
```

Comment:
!!! This will not work in Flatpak. This deserves its own Vinegar issue - "DXVK required by Proton and cannot be disabled, but renderer can change in Studio"

For now,
```suggestion
Change the Roblox Studio renderer from Studio's settings to Vulkan
```